app/assets/javascripts/organizations.js

```diff
@@ -0,0 +1,10 @@
+$(function () {
+
+  "use strict";
+
+  $(".manually_create").on("click", function () {
```

Comment:
@dissident I think you may use collapse feature from bootstrap for this http://getbootstrap.com/javascript/#collapse